### PR TITLE
github: add issue and pull request templates

### LINKED ISSUE #405: github: add issues template

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

Closes #405.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### What did you expect to happen?
+
+
+### What actually happened?
+
+
+### What version of Gonum are you using?
+
+<!-- If you are installing from source paste the output of `(cd $(go env GOPATH)/src/gonum.org/v1/gonum && git rev-parse HEAD)`. -->
```
Comment (sbinet) [inline .github/ISSUE_TEMPLATE]: perhaps also ask for the Go version?

Comment (kortschak) [reply .github/ISSUE_TEMPLATE]: Good idea - done

**Comment by kortschak** [comment]
PTAL

**Comment by btracey** [comment]
I wonder if we should have a comment about questions going to gonum-dev

**Comment by kortschak** [comment]
That sounds like a good idea.

**Pull request merged**